docs/standards/generic-standards/lsp23-linked-contracts-factory.md

```diff
@@ -0,0 +1,90 @@
+---
+sidebar_label: 'LSP23 - Linked Contracts Factory'
+sidebar_position: 7
+---
+
+# LSP23 Linked Contracts Factory
+
+:::info Standard Document
+
+[LSP23 - Linked Contracts Factory](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-23-LinkedContractsFactory.md)
+
+:::
+
+## Introduction
+
+Deploying smart contracts that are interdependent can be a complex process. Often, two or more contracts need each other's addresses at deployment, creating a circular dependency that can be challenging to resolve.
+
+![Circular Depedency](/img/standards/lsp23/circular-dependency.jpeg)
+
+The LSP23 Linked Contracts Factory addresses this issue by providing a unified interface for deploying linked contracts with an optional post-deployment execution. This standard simplifies the deployment of complex contract systems, making it easier for developers to deploy interdependent contracts without the need for manual intervention.
+
+## What does this standard represent?
+
+This standard represents a smart contract factory that allows users to deploy primary and secondary contracts that are linked together. It also supports deploying contracts as `ERC1167` proxies. The factory can execute post-deployment modules to perform additional logic after deploying the contracts (for example, it could pass the address of the secondary to the primary contract).
```

Comment:
Not sure about the suggested changes for the `constructor` / initializer function.
Otherwise, maybe it would be good to define what is:
- the **primary** contract
- the **secondary** contract

```suggestion
This standard represents a smart contract factory that allows users to deploy **primary** and **secondary** contracts that are linked together. It also supports deploying contracts as `ERC1167` minimal proxies. The factory can execute post-deployment modules to perform additional logic after deploying the contracts (_example: pass the address of the secondary contract to the primary contract's `constructor` / initializer function_).
```